core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGeneData.java

```diff
@@ -527,24 +597,30 @@ public static boolean updateLength(String symbol, String chromosome, List<long[]
         ReferenceGenomeGene refGene = daoReferenceGenomeGene.getGene(gene.getEntrezGeneId(), refreneceGenomeId);
         /// If it's not in the database, don't add it
         if (refGene != null) {
-                /// Calc length
+            /// Calc length
             long[] exonic = calculateGeneLength(loci);
-                
+
             /// If there is a cytoband in database, check if cytoband-chr matches input-chr
-            String cbChr = "chr"+refGene.getCytoband();
```

Comment:
I believe I am fixing a bug here.  See: https://github.com/cBioPortal/cbioportal/commit/b1b65c78668ebdfbb9d6921ccade66d170c04025#diff-ce623038e7b8c7a5b69d28130fe0729842482421327b51e6caf583deb8af4b30R518
```
String cbChr = "chr"+refGene.getCytoband();
System.out.println(" updateLength--> cytoband of the gene:  " + cbChr);
if (cbChr.equals(chromosome)) { //Update the length only if the chromosome matches
```
Example cytobands are "16q22.1" and "19q13|19q13-qter". An example chromosome in the gtf file (which is what chromosome comes from) is "chr19". Obviously "chr" + "19q13|19q13-qter" will never equal "chr19". You can see the code had been
```
String cbChr = "chr"+cytoband.split("p|q")[0];
```
Which makes more sense to me.